packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -1114,7 +1114,7 @@ describe('I18n', () => {
           'humanize.weekday',
           {
             pseudotranslate: false,
-            replacements: {day: 'Friday', time: '11:00 a.m.'},
+            replacements: {day: 'Saturday', time: '11:00 a.m.'},
```

Comment:
I dont quite understand what this is fixing, do you have an original issue that I can take a look?

Reply:
Reverting a prior fix: https://github.com/Shopify/quilt/pull/916#pullrequestreview-285701811, which changed it to "Friday" as for some reason that was needed to get the test to pass in that PR, but prior to that it was "Saturday" (and _it should be_ "Saturday") as that is the correct weekday for 2012-12-15T00:00:00-00:00 GMT in Sydney, Australia.